src/test/java/seedu/address/model/person/PersonTest.java

```diff
@@ -44,11 +44,6 @@ public void isSamePerson() {
         // name differs in case, all other attributes same -> returns false
         Person editedBob = new PersonBuilder(BOB).withName(VALID_NAME_BOB.toLowerCase()).build();
         assertFalse(BOB.isSamePerson(editedBob));
-
-        // name has trailing spaces, all other attributes same -> returns false
```

Comment:
Why was this test deleted?

Reply:
The updated regex now doesn't allow trailing spaces. Actually such cases will never happen because the parser trim the string

Reply:
Ok sure, I think you can delete it then?